### PR TITLE
stop parser with script, give time to Custom Elements to upgrade.

### LINKED ISSUE #505: `document.getElementById(...).create` is not a function
Both the `create` and `restore` methods of `test-fixture` might be `undefined` at the moment a test runs because the element might not be upgraded. [See this travis build](https://travis-ci.org/PolymerElements/iron-collapse/builds/208777695). The tests are usually very flaky in M57.

While debugging wct /w @valdrinkoshi, we found that adding a script like `<script>void(0)</script>` before `<test-fixture>` pauses the parser until the javascript for the imports has run.  Also adding the HTML import before the `web-component-tester/browser.js` import fixes the issue.

This is again the ordering problem.

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Fixes #505, more info on the bug here https://bugs.chromium.org/p/chromium/issues/detail?id=701601

 - [x] CHANGELOG.md has been updated

### CODE REVIEW

File: browser/environment.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ export function loadSync() {
     // Synchronous load.
     document.write('<link rel="import" href="' + encodeURI(url) + '">'); // jshint ignore:line
   });
+  if (imports.length) {
```
Comment (rictic) [inline browser/environment.js]: Prefer the more explicit `if (imports.length > 0) {`

Comment (valdrinkoshi) [inline browser/environment.js]: 👌 done!

**Comment by rictic** [comment]
LGTM, good to merge

**Comment by paladox** [comment]
Could this fix be back ported to 5.x release please?

We are getting the same problem on https://gerrit-ci.gerritforge.com/job/Gerrit-bazel-stable-2.14/24/console

**Pull request merged**